### PR TITLE
CRUD operations for products

### PR DESCRIPTION
**Pull request by alekvan**
New endpoint handlers for products route

### CODE REVIEW

File: social-app/app.js [hunk 1]
```diff
@@ -7,9 +7,10 @@ const postsRouter = require('./routes/posts');
 const usersRouter = require('./routes/users');
 const commentsRouter = require('./routes/comments');
 const paymentsRouter = require('./routes/payments');
+const productsRouter = require('./routes/products');
 const mongoose = require('mongoose');
 
-require('dotenv').config();
+// require('dotenv').config();
```
Comment (bobansugareski) [inline social-app/app.js:13]: Unnecessary line

File: social-app/models/products.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    creator: {
+      type: mongoose.Types.ObjectId,
+      ref: 'user',
+    },
+  },
+  { timestamps: true }
+);
+
+module.exports = mongoose.model('products', productsSchema);
```
Comment (bobansugareski) [inline social-app/models/products.js:29]:
```suggestion
module.exports = mongoose.model('product', productsSchema);
```

**Review by bobansugareski — CHANGES_REQUESTED** [review]

File: social-app/controllers/products.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const Product = require('../models/products');
```
Comment (bobansugareski) [inline social-app/controllers/products.js:1]:
```suggestion
const Product = require('../models/product');
```

File: social-app/app.js (hunk 1 above)
Comment (bobansugareski) [reply social-app/app.js:13]: Revert the change here

Comment (alekvan) [reply social-app/app.js]: Fixed

File: social-app/controllers/products.js (hunk 3 above)
Comment (alekvan) [reply social-app/controllers/products.js]: Fixed

File: social-app/models/products.js (hunk 2 above)
Comment (alekvan) [reply social-app/models/products.js]: Fixed

**Review by bobansugareski — APPROVED** [review]

**Review by stefanaleksovski1999 — APPROVED** [review]

**Pull request merged**